src/core/client/ui/components/Button/Button.css

```diff
@@ -573,3 +580,33 @@
     }
   }
 }
+
+.variantTextUnderlined {
+  font-weight: 700;
+  &:not(.disabled) {
+    &.colorRegular {
+      &.mouseHover {
+        color: var(--palette-primary-main)!important;
```

Comment:
Is the  `!important` here really necessary? My quick tests seems to work also without it?